specs/v2-sketch.yaml

```diff
@@ -282,40 +264,29 @@ components:
               student:
                 type: boolean
                 default: false
-    Organization:
-      description: "NOTE: The definition of Organization is subject to change."
-      type: object
-      properties:
-        id:
-          type: string
-          format: uuid
     Project:
```

Comment:
Do you want to separately add things like the project name, and maybe some other info like date created, who added it, etc? Thinking through what a developer might want to know when querying a project. Would be curious to know what's "cheap" for us to have available here.

Reply:
The cheapest are:
* project name
* vcs info
    - type (GitHub/Bitbucket/etc.)
    - master branch
    - link back to repo on the vcs provider
* last job finish time
* the number for the next job to start (1 greater than the most recent job)
* parallelism
* the names (not values) of any env-vars (I don't think that data belongs in this response anyway)

Data that would involve additional DB accesses:
* user who added the project
* full list of users following the project (again, I don't think that belongs here)

Reply:
Let's do project name, vcs info, and last job finish time. The others feel superfluous. We can punt on user who added or anything about followers.